AnnotationPreprocessing/AnnotationPreprocessing.nf

```diff
@@ -120,7 +120,8 @@ process busco {
         cp -r \$AUGUSTUS_CONFIG_PATH/* \$AUG_CONF_DIR
         export AUGUSTUS_CONFIG_PATH=\$AUG_CONF_DIR
     fi
-    echo "BUSCO_CONFIG_FILE=\$BUSCO_CONFIG_FILE"
+    # before with buscov4 it was echo "BUSCO_CONFIG_FILE=\$BUSCO_CONFIG_FILE", it stops working for buscov5
```

Comment:
This is the kind of comment I would just leave in the commit message. One of the great things with version control is to move this kind of information out of the code.